src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -325,9 +325,6 @@ public int MaxModelBindingRecursionDepth
         /// </summary>
         public JsonSerializerOptions SerializerOptions { get; } = new JsonSerializerOptions
         {
-            // Allow for the payload to have null values for some inputs (under-binding)
-            IgnoreNullPropertyValueOnRead = true,
```

Comment:
None of the tests failed, removing this to unblock future intake of newer runtime